tasks/bash.sh

```diff
@@ -25,18 +25,37 @@ success() {
   echo "$1"
 }
 
+determine_command_for_facter_4() {
+  puppet_version="$(puppet --version)"
+
+  if (( ${puppet_version%%.*} == 6 )); then
+    # puppet 6 with facter 4
+    facts_command="facter --json --show-legacy"
```

Comment:
Could we change all of the `facts_command` assignments to an array?  i.e.

```
facts_command=(facter --json --show-legacy)
```

That will prevent any quoting or word splitting issues in the `exec`.

Reply:
@m0dular nice suggestion, changed to array.